

drivers/scsi/psi240i.c:713: unknown field `abort' specified in initializer
drivers/scsi/psi240i.c:713: warning: initialization from incompatible pointer type
drivers/scsi/psi240i.c:714: unknown field `reset' specified in initializer
drivers/scsi/psi240i.c:714: warning: initialization from incompatible pointer typ

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/scsi/psi240i.c |   37 -------------------------------------
 1 files changed, 37 deletions(-)

diff -puN drivers/scsi/psi240i.c~psi240i-build-fix drivers/scsi/psi240i.c
--- 25/drivers/scsi/psi240i.c~psi240i-build-fix	2004-10-05 16:20:09.639109048 -0700
+++ 25-akpm/drivers/scsi/psi240i.c	2004-10-05 16:20:31.846732976 -0700
@@ -642,41 +642,6 @@ static int Psi240i_Release(struct Scsi_H
 }
 
 /****************************************************************
- *	Name:	Psi240i_Abort
- *
- *	Description:	Process the Abort command from the SCSI manager.
- *
- *	Parameters:		SCpnt - Pointer to SCSI command structure.
- *
- *	Returns:		Allways snooze.
- *
- ****************************************************************/
-int Psi240i_Abort (Scsi_Cmnd *SCpnt)
-	{
-	DEB (printk ("psi240i_abort\n"));
-	return SCSI_ABORT_SNOOZE;
-	}
-/****************************************************************
- *	Name:	Psi240i_Reset
- *
- *	Description:	Process the Reset command from the SCSI manager.
- *
- *	Parameters:		SCpnt - Pointer to SCSI command structure.
- *					flags - Flags about the reset command
- *
- *	Returns:		No active command at this time, so this means
- *					that each time we got some kind of response the
- *					last time through.  Tell the mid-level code to
- *					request sense information in order to decide what
- *					to do next.
- *
- ****************************************************************/
-int Psi240i_Reset (Scsi_Cmnd *SCpnt, unsigned int reset_flags)
-	{
-	return SCSI_RESET_PUNT;
-	}
-
-/****************************************************************
  *	Name:	Psi240i_BiosParam
  *
  *	Description:	Process the biosparam request from the SCSI manager to
@@ -710,8 +675,6 @@ static Scsi_Host_Template driver_templat
 	.detect			= Psi240i_Detect,
 	.release		= Psi240i_Release,
 	.queuecommand		= Psi240i_QueueCommand,
-	.abort	  		= Psi240i_Abort,
-	.reset	  		= Psi240i_Reset,
 	.bios_param	  	= Psi240i_BiosParam,
 	.can_queue	  	= 1,
 	.this_id	  	= -1,
_
